ckan/plugins/toolkit.py

```diff
@@ -141,7 +141,7 @@ def _initialize(self):
 current request.
 
 '''
-        t['h'] = getattr(pylons.config['pylons.h'], 'no_magic', None)
+        t['h'] = pylons.config['pylons.h']
```

Comment:
Yes, we can do this since it's only ever modified in-place. I will make this change in the morning.